content/src/main/content/jcr_root/apps/acs-commons/config.author/org.apache.sling.commons.log.LogManager.factory.config-acssample.config

```diff
@@ -0,0 +1,5 @@
+# Configuration created by Apache Sling JCR Installer
```

Comment:
Why is this necessary?

Reply:
Committed by mistake. I have to delete this file.